CHANGELOG.md

```diff
@@ -3,14 +3,18 @@ SPDX-FileCopyrightText: 2009 Fermi Research Alliance, LLC
 SPDX-License-Identifier: Apache-2.0
 -->
 
-## Changes Since Last Release OR vX.Y.Z \[yyyy-mm-dd\]
+## v3.10.11 \[2025-03-24\]
 
-Changes since the last release
+stale_age allows to keep Glideins in queues for longer than one week.
+Apptainer test now may be successful even if the default image is not available.
+Check the changed defaults, including SINGULARITY_IMAGE_REQUIRED, APPTAINER_TEST_IMAGE and the redirection to https for monitoring pages.
```

Comment:
Do these lines fall under a particular section? I see that the line `Changes since the last release` has been removed which was the previous version.

Reply:
These are general notes about this release. "Changes since the last release" is a placeholder from the template